extensions/tiled-assets/examples/example-dimension.json

```diff
@@ -0,0 +1,234 @@
+            "hrefTemplate": "http://cloudless.eox.at/sen2l2a-cog-converted/{timeSpan}_s2_l2a/{timeSpan}_s2_l2a_B11.tif",
+            "title": "",
+            "type": "image/vnd.stac.geotiff",
+            "eo:bands": [
+                7
+            ]
+        }
+    },
+    "assets": {
+        "B02": {
+            "hrefTemplate": "http://cloudless.eox.at/sen2l2a-cog-converted/2018-02-26_2018-03-11_s2_l2a/2018-02-26_2018-03-11_s2_l2a_B02.tif",
+            "title": "",
+            "type": "image/vnd.stac.geotiff",
+            "eo:bands": [
+                0
+            ]
+        },
+        "B03": {
+            "hrefTemplate": "http://cloudless.eox.at/sen2l2a-cog-converted/2018-02-26_2018-03-11_s2_l2a/2018-02-26_2018-03-11_s2_l2a_B03.tif",
+            "title": "",
+            "type": "image/vnd.stac.geotiff",
+            "eo:bands": [
+                1
+            ]
+        },
+        "B04": {
+            "hrefTemplate": "http://cloudless.eox.at/sen2l2a-cog-converted/2018-02-26_2018-03-11_s2_l2a/2018-02-26_2018-03-11_s2_l2a_B04.tif",
+            "title": "",
+            "type": "image/vnd.stac.geotiff",
+            "eo:bands": [
+                2
+            ]
+        },
+        "B05": {
+            "hrefTemplate": "http://cloudless.eox.at/sen2l2a-cog-converted/2018-02-26_2018-03-11_s2_l2a/2018-02-26_2018-03-11_s2_l2a_B05.tif",
+            "title": "",
+            "type": "image/vnd.stac.geotiff",
+            "eo:bands": [
+                3
+            ]
+        },
+        "B06": {
+            "hrefTemplate": "http://cloudless.eox.at/sen2l2a-cog-converted/2018-02-26_2018-03-11_s2_l2a/2018-02-26_2018-03-11_s2_l2a_B06.tif",
+            "title": "",
+            "type": "image/vnd.stac.geotiff",
+            "eo:bands": [
+                4
+            ]
+        },
+        "B07": {
+            "hrefTemplate": "http://cloudless.eox.at/sen2l2a-cog-converted/2018-02-26_2018-03-11_s2_l2a/2018-02-26_2018-03-11_s2_l2a_B07.tif",
+            "title": "",
+            "type": "image/vnd.stac.geotiff",
+            "eo:bands": [
+                5
+            ]
+        },
+        "B08": {
+            "hrefTemplate": "http://cloudless.eox.at/sen2l2a-cog-converted/2018-02-26_2018-03-11_s2_l2a/2018-02-26_2018-03-11_s2_l2a_B08.tif",
+            "title": "",
+            "type": "image/vnd.stac.geotiff",
+            "eo:bands": [
+                6
+            ]
+        },
+        "B11": {
+            "hrefTemplate": "http://cloudless.eox.at/sen2l2a-cog-converted/2018-02-26_2018-03-11_s2_l2a/2018-02-26_2018-03-11_s2_l2a_B11.tif",
+            "title": "",
+            "type": "image/vnd.stac.geotiff",
+            "eo:bands": [
+                7
+            ]
+        }
+    },
+    "links": [
+        {
+            "rel": "self",
+            "href": "http://cloudless.eox.at/sen2l2a-cog-converted.json"
+        },
+        {
+            "rel": "root",
+            "href": "http://cloudless.eox.at/stack.json"
```

Comment:
Very minor: I guess that was meant to be named stac instead of stack ;-)

Reply:
👍